### ISSUE TITLE
Support for Gaussian splats / .ply files ?

### ISSUE THREAD

**Issue by cs50victor** [simplerr/rust-renderer#9]
Hey @simplerr , really love your on the rust-renderer.

It's one of the main inspiration behind a simpler ( Vulkan to Webrtc ) renderer I've been working on github.com/cs50victor/kitt2

I was wondering if there's any plan to add support for Gaussian splats (.ply files), if so how could I help contribute?

**Comment by simplerr** [comment]
Hello, thanks for checking out this project and I'm happy to hear it even has been inspiration for your own project! 

Currently I don't have any plans to add Gaussian splat support but they seem very interesting and I would like to learn more about them in the future. I know too little about the techniques used when rendering gaussians but I guess the (somewhat) easy to use render graph system in rust-renderer could be helpful in creating a small prototype for it but I also think it's too immature and would need a lot of changes in the core renderer files due to not having exposed everything to the render graph system.

Creating a new standalone sample that renders Gaussians is something I absolutely could merge to the main branch without much thought but if there are as many changes to the core rendering code I suspect then it'll take more time to discuss and review and I don't have that right now.

I'd say there are a lot better options for other renderers than this project if you want to tinker with Gaussians but if you really want to feel free 😃

**Comment by cs50victor** [comment]
fair point. Currently also looking into / contributing to https://github.com/mosure/bevy_gaussian_splatting . Hopefully when the field matures and we have a somewhat stable renderer using wgpu i can help port it to vulkan using the vulkano crate.
